app-admin/cdist/cdist-6.9.2.ebuild

```diff
@@ -3,20 +3,28 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+PYTHON_COMPAT=( python3_{6..9} )
 DISTUTILS_USE_SETUPTOOLS=no
 
 inherit distutils-r1
 
 DESCRIPTION="A usable configuration management system"
 HOMEPAGE="https://www.cdi.st/ https://code.ungleich.ch/ungleich-public/cdist"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+SRC_URI="https://code.ungleich.ch/ungleich-public/cdist/-/archive/${PV}/cdist-${PV}.tar.gz"
 
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="doc"
 
+distutils_enable_tests unittest
+
+python_prepare_all() {
+	echo "VERSION='${PV}'" > "${S}"/cdist/version.py
```

Comment:
`echo "VERSION='${PV}'" > cdist/version.py || die`
1. you're doing I/O with an external command, which always requires a `|| die`
2. you're already in `${S}`, hence `"${S}"/` is unnecessary

Reply:
Thanks, fixed :smiley: